src/pages/plan/utils/MemoDispatcherResolver.ts

```
@@ -0,0 +1,36 @@
+  MemoInitType,
+  MemoResponseType,
+  MemoUpdateType,
+} from '../types/MemoResponseBodyType';
+
+export function MemoDispatcherResolver(message: any) {
+  const memoData: MemoResponseType = JSON.parse(message.body);
+  console.log('MEMO 메시지:', memoData);
+  switch (memoData.type) {
+    case 'INIT':
+      const memoInit = memoData as MemoInitType;
+      for (let i = 0; i < memoInit.MEMO.length; i++) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)